sympy/utilities/tests/test_lambdify.py

```diff
@@ -257,6 +257,13 @@ def test_issue_9334():
     foo, bar = numpy.random.random((2, 4))
     func_numexpr(foo, bar)
 
+def test_issue_12984():
```

Comment:
Please rename this test to a legible name that captures the essence of the fix

Reply:
I don't like `test_issue_1234` but it is widely used. I think it makes sense when the test is for something so obscure that only the issue number adequately explains it.

I think this is fine for this PR since it is currently standard but in general it would be better to use meaningful names where possible and have links or issue numbers in the comments.

Reply:
@oscarbenjamin Sure, we can have that change.